sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -509,7 +509,7 @@ static int sof_audio_probe(struct platform_device *pdev)
 		/* default quirk for legacy cpu */
 		sof_rt5682_quirk = SOF_RT5682_SSP_CODEC(2);
 	} else {
-		dmic_num = 1;
+		dmic_num = 2;
```

Comment:
We probably need to rename 'dmic_num' as 'dmic_dailink_num' since this is completely misleading.

Reply:
OK, let me change it to dmic_be_num?